api.py

```diff
@@ -0,0 +1,76 @@
+import os
+import logging
+import requests
+import flask
+import datetime
+
+
+from flask import Flask, request, flash, redirect, render_template, url_for, jsonify
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+app.config.update(dict(
+    SQLALCHEMY_DATABASE_URI='sqlite:///kraken.db',
+    DEBUG=True,
+    SECRET_KEY='development',
+    LOG_LEVEL=logging.DEBUG,
+    ENVIRONMENT='development'
+))
+app.config.from_envvar('FLASK_SETTINGS', silent=True)
+
+logging.basicConfig(level=logging.DEBUG, filename='logs/' + app.config['ENVIRONMENT'] + '.log')
+
+db = SQLAlchemy(app)
+
+# TODO: reused model from sesh-dash! DRY!!! - at least move database stuff it into a separate file
+class Sesh_Site(db.Model):
+    __tablename__= 'sites'
+    id = db.Column(db.Integer, primary_key=True)
+    site_name = db.Column(db.String(100))
+
+    def __str__(self):
+        return self.site_name
+
+class BoM_Data_Point(db.Model):
```

Comment:
I would not use a submodule to import the whole sesh-dash repo just to simply import a file. Not sure what is the best way to do it. (that's why I ended up adding the classes here) 
I'd somehow say we should extract the models into its own package (but only if we will use in other apps, too. This is quite an effort I guess)

Besides the duplication I did not get the django models working here in flask. So right now we can not reuse the same models because of the django dependency. Do you have an idea on that? is it possible to use the django ORM without django? any hints on that? 

Another option which I did not get working so far is to inspect the columns of the tables and let the ORM define the methods. This would at least minimize the code and would not require us to change the code if we update the tables. 
Because this API is mainly for dumping data I think this could be a compromise.